### PR TITLE
Bundle the updater when bumping versions

### PR DESCRIPTION
**Pull request by brrygrdn**
This updates the bin/bump-versions.rb script to run `bundle` in the Updater directory to ensure the Gemfile.lock picks up the new version as part of the staged changes.

### Notes

This does assume the host machine has ruby/bundler installed and is ok bundling the updater gems locally but on the upside it verifies the bundling goes as expected before the changes leave the machine.

### CODE REVIEW

**Review by jakecoffman — APPROVED** [review]

**Comment by deivid-rodriguez** [comment]
Alternatively `bundle lock` (just brings lockfile is sync but does not install gems) may work?

**Comment by brrygrdn** [comment]
> Alternatively bundle lock (just brings lockfile is sync but does not install gems) may work?

Aha, yes, that's much better, I always forget that exists!

File: bin/bump-version.rb [hunk 1]
```diff
@@ -75,6 +75,9 @@ def proposed_changes(version, _new_version)
   File.write(version_path, new_version_contents)
   puts "☑️  common/lib/dependabot.rb updated"
 
+  # Bump the updater's Gemfile with the new version
```
Comment (deivid-rodriguez) [inline bin/bump-version.rb:78]:
```suggestion
  # Bump the updater's Gemfile.lock with the new version
```

**Review by deivid-rodriguez — APPROVED** [review]
Nice!

**Pull request merged**